### PR TITLE
FOUNDATIONS: Retrieve Team By Id

### LINKED ISSUE #89: FOUNDATIONS: Retrieve Team By Id

### PR DESCRIPTION
**Pull request by jasurbekyusuf**
Closes #89

### CODE REVIEW

File: Tarteeb.Api.Tests.Unit/Services/Foundations/Teams/TeamServiceTests.Exceptions.RetrieveById.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿//=================================
+// Copyright (c) Coalition of Good-Hearted Engineers
+// Free to use to bring order in your workplace
+//=================================
+
+using Microsoft.Data.SqlClient;
+using Moq;
+using System.Threading.Tasks;
+using System;
```
Comment (jamshidbek001) [inline Tarteeb.Api.Tests.Unit/Services/Foundations/Teams/TeamServiceTests.Exceptions.RetrieveById.cs:9]: order?

File: Tarteeb.Api/Services/Foundations/Teams/TeamService.cs [hunk 2]
```diff
@@ -34,5 +36,19 @@ public partial class TeamService : ITeamService
...
+            ValidateTeamId(teamId);
+
+            Team maybeTeam =
+                await storageBroker.SelectTeamByIdAsync(teamId);
+
+            ValidateStorageTeam(maybeTeam, teamId);
+
+            return maybeTeam;
+        });
+
```
Comment (jamshidbek001) [inline Tarteeb.Api/Services/Foundations/Teams/TeamService.cs:52]: please remove empty line

**Review by jamshidbek001 — CHANGES_REQUESTED** [review]
order usings

**Review by jamshidbek001 — APPROVED** [review]
LGTM

**Review by ElbekDeveloper — APPROVED** [review]

**Pull request merged**